src/utils/fxa.ts

```diff
@@ -386,7 +386,10 @@ async function applyCoupon(
     }
   } catch (e) {
     if (e instanceof Error) {
-      logger.error("apply_fxa_coupon", { stack: e.stack, message: e.message });
+      logger.error(
+        "apply_fxa_coupon",
```

Comment:
Logs aren't displaying this properly

Reply:
Hm yeah we should figure this out, lgtm for now